dimensiondata/pom.xml

```diff
@@ -144,6 +144,7 @@
                                         <test.dimensiondata-cloudcontrol.credential>
                                             ${test.dimensiondata-cloudcontrol.credential}
                                         </test.dimensiondata-cloudcontrol.credential>
+                                        <jclouds.zones>LAB04_N2_VMWARE_1</jclouds.zones>
```

Comment:
We'll probably have to have a closer look at this when implementing the compute service, but it is OK for now.

Just FTR, but not to have it fixed now, the compute service has the `listAssignableLocations` method that returns the locations (zones, as per previous discussions) where users can deploy VMs. jclouds will then cache those locations in a memoized supplier. Having HTTP filters such as the DataCenterId filter that have location suppliers injected, may cause infinite loops if the `listAssignableLocations` method in the compute service calls api methods with those filters.

This is not something we need to fix now, but to keep in mind when implementing the compute service, and for a possible upcoming discussion about how to properly model the locations for this provider.